### PR TITLE
Fix builds

### PR DESCRIPTION
**Pull request by tolbrino**
I've created a separate issue #12 to track the present shortcoming.

### CODE REVIEW

**Review by AleksandarFilipov — DISMISSED** [review]
works!

**Comment by AleksandarFilipov** [comment]
So basically `make clean deps compile` works great now. Not sure why. Lets update the doc again.

File: Makefile [hunk 1]
```diff
@@ -12,12 +12,16 @@ all: help
 .PHONY: deps
 deps: prepare
 deps: ## Get and compile Elixir dependencies
-	$(mix) do deps.get, deps.compile
+	$(mix) deps.get
```
Comment (AleksandarFilipov) [inline Makefile:15]: So this seems to be the thing that fixed the broken thing.

File: Makefile [hunk 2]
```diff
@@ -12,12 +12,16 @@ all: help
...
 
 .PHONY: compile
-compile: ## Compile Elixir code
+compile: ## BROKEN: Compile Elixir code
```
Comment (AleksandarFilipov) [inline Makefile]: Isn't broken whan i run it....

**Comment by tolbrino** [comment]
It only works after you've started the shell once, because then all deps were compiled. Generally I would consider it to be broken still.

**Comment by AleksandarFilipov** [comment]
Clone followed by,
make clean deps compile 
works. No shell start in between; give it a try!

**Comment by tolbrino** [comment]
> Clone followed by,
> make clean deps compile
> works. No shell start in between; give it a try!

Agreed. It works. But that is by chance. Neither `compile` nor `deps.compile` checks the dependency tree properly, resulting in different behaviour were one succeeds by accident.

**Review by AleksandarFilipov — APPROVED** [review]

**Pull request merged**